srf_generation/source_parameter_generation/uncertainties/mag_scaling.py

```diff
@@ -1,7 +1,8 @@
 from enum import Enum
 from typing import Union
 
-import numpy as np
+from magnitude_scaling.allen_2017 import *
+from magnitude_scaling.strasser_2010 import *
```

Comment:
import * ?
is this avoidable?
or is this a design decision?

Reply:
It's a mix of me being lazy, and also keeping the allen and strasser mwsr's consistent with the other ones locally in this file.
Changing it to be "from magnitude_scaling import allen, strasser" is ok too if that's the style we want

Reply:
If doing this edit wont cause any cascading effect, i would prefer it to be changed, as it will avoid namespace conflicts and easier to track if error accures. 
But if it's too much of a trouble then we should fix it later.